### ISSUE TITLE
v1.1.1 doesn't build - Missing qatzip.pc.in

### ISSUE THREAD

**Issue by tj90241** [intel/QATzip#71]
I think a file was missed in #67? There is no `qatzip.pc.in` anywhere in this repository...

```
$ ./autogen.sh 
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force 
autoreconf: configure.ac: tracing
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
autoreconf: running: /usr/bin/autoconf --force
autoreconf: configure.ac: not using Autoheader
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:53: installing './ar-lib'
configure.ac:43: installing './compile'
configure.ac:57: installing './config.guess'
configure.ac:57: installing './config.sub'
configure.ac:40: installing './install-sh'
configure.ac:40: installing './missing'
configure.ac:287: error: required file 'qatzip.pc.in' not found
```

**Comment by tj90241** [comment]
I'm not sure what happened, but the commit in the main branch:
https://github.com/intel/QATzip/pull/70/commits/4a67fac2fe8e5bfc0a4e48c67e77d6b6f589c712

Does *not* have the file...

But PR#67 does:
https://github.com/intel/QATzip/pull/67/commits/db1f79f458bd8cbf5732b32ef148f19a6e335267

**Comment by XingHongChenIntel** [comment]
yes, we're trying to fix this issue, very thanks to raise this issue.

**Comment by tj90241** [comment]
Thanks for the fast response!

FYI, the tag v1.1.1 looks force-pushed with this fix, but now GitHub shows latest release as v1.1.0?
https://github.com/intel/QATzip/releases

**Comment by XingHongChenIntel** [comment]
I have remove the old tag and release, Now, the fixed release v1.1.1 have drafted, please review, thanks for your comments.

**Comment by tj90241** [comment]
Looks great and my CI is passing again, thanks!
